openrc-designate.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+export OS_USERNAME=admin
+export OS_USER_DOMAIN_NAME=default
+export OS_PROJECT_NAME=admin
+export OS_PROJECT_DOMAIN_NAME=default
+export OS_IDENTITY_API_VERSION=3
+export OS_AUTH_URL=http://centos-sentmwlu.example-service-dev.al.vps.tecnico.ulisboa.pt:5000/v3
```

Comment:
Again, the URL of the VPS.

Reply:
We need to test against something, what we have at our disposal is a VM in our VPS.

This is also related to the discussion below. If we change the VM we are testing against then we might also need to change the default quotas.

Reply:
Why don't we include these links in some private repo or something? Sounds weird to opensource these links...